bezier-rs/lib/src/lib.rs

```diff
@@ -721,6 +699,141 @@ impl Bezier {
 		});
 		result
 	}
+
+	/// Approximate a bezier curve with circular arcs.
+	/// The algorithm can be customized using the [ArcsOptions] structure.
+	pub fn arcs(&self, arcs_options: ArcsOptions) -> Vec<CircleArc> {
+		let ArcsOptions { maximize_arcs, error, max_iterations } = arcs_options;
+		if maximize_arcs {
+			return self.arcs_helper(0., 1., error, max_iterations);
+		}
+		let mut extrema: Vec<f64> = self.local_extrema().into_iter().flatten().collect::<Vec<f64>>();
+		extrema.append(&mut vec![0., 1.]);
+		extrema.dedup();
+		extrema.sort_by(|ex1, ex2| ex1.partial_cmp(ex2).unwrap());
```

Comment:
I'm assuming you mean reduce? Also, what's the benefit of making the helper public? I.e. would users care about this functionality/when would they use it?

Reply:
Oops, yes, I meant reduce. I remember that bezier.rs returned a third argument (in addition to the x and y extrema) which was a list of both x and y extrema with duplicates removed. Created a function to handle output of that form could be valuable. But, you could always just make it a private helper for now because making it public is easily something that could be done in the future if there a need.